pkg/sdk/warehouse.go

```diff
@@ -71,22 +71,21 @@ type WarehouseCreateOptions struct {
 	// Object properties
 	WarehouseType                   *WarehouseType `ddl:"parameter,single_quotes" db:"WAREHOUSE_TYPE"`
 	WarehouseSize                   *WarehouseSize `ddl:"parameter,single_quotes" db:"WAREHOUSE_SIZE"`
-	MaxClusterCount                 *uint8         `ddl:"parameter" db:"MAX_CLUSTER_COUNT"`
-	MinClusterCount                 *uint8         `ddl:"parameter" db:"MIN_CLUSTER_COUNT"`
+	MaxClusterCount                 *int           `ddl:"parameter" db:"MAX_CLUSTER_COUNT"`
+	MinClusterCount                 *int           `ddl:"parameter" db:"MIN_CLUSTER_COUNT"`
 	ScalingPolicy                   *ScalingPolicy `ddl:"parameter,single_quotes" db:"SCALING_POLICY"`
-	AutoSuspend                     *uint          `ddl:"parameter" db:"AUTO_SUSPEND"`
+	AutoSuspend                     *int           `ddl:"parameter" db:"AUTO_SUSPEND"`
 	AutoResume                      *bool          `ddl:"parameter" db:"AUTO_RESUME"`
 	InitiallySuspended              *bool          `ddl:"parameter" db:"INITIALLY_SUSPENDED"`
 	ResourceMonitor                 *string        `ddl:"parameter,double_quotes" db:"RESOURCE_MONITOR"`
 	Comment                         *string        `ddl:"parameter,single_quotes" db:"COMMENT"`
 	EnableQueryAcceleration         *bool          `ddl:"parameter" db:"ENABLE_QUERY_ACCELERATION"`
-	QueryAccelerationMaxScaleFactor *uint8         `ddl:"parameter" db:"QUERY_ACCELERATION_MAX_SCALE_FACTOR"`
+	QueryAccelerationMaxScaleFactor *int           `ddl:"parameter" db:"QUERY_ACCELERATION_MAX_SCALE_FACTOR"`
 
 	// Object params
-	MaxConcurrencyLevel             *uint            `ddl:"parameter" db:"MAX_CONCURRENCY_LEVEL"`
-	StatementQueuedTimeoutInSeconds *uint            `ddl:"parameter" db:"STATEMENT_QUEUED_TIMEOUT_IN_SECONDS"`
-	StatementTimeoutInSeconds       *uint            `ddl:"parameter" db:"STATEMENT_TIMEOUT_IN_SECONDS"`
-	Tags                            []TagAssociation `ddl:"list,parentheses" db:"TAG"`
```

Comment:
Why remove the Tags field? we won't use it in the Terraform provider but the SDK should probably still support this operation.

Reply:
Tags are not parts of the create operation, they are part of alter

Reply:
Yes they are. :)
https://docs.snowflake.com/en/sql-reference/sql/create-warehouse
![image](https://github.com/Snowflake-Labs/terraform-provider-snowflake/assets/128428373/fe7828fd-757c-40a3-9602-78f442a24983)
I was even able to use that in the "unset tags" integration tests.